src/peft/peft_model.py

```diff
@@ -145,7 +145,7 @@ def save_pretrained(
         save_directory: str,
         safe_serialization: bool = False,
         selected_adapters: Optional[List[str]] = None,
-        save_function = torch.save,
+        save_function: Optional[function] = None,
```

Comment:
The type signature is not correct, it would have to be something like:

`Optional[Callable[[Dict[str, torch.tensor], str], None]]`

if I'm not missing something (we don't actually check the types, it's really just for documentation at this point).